encoding/masking-limiting-etc.md

```diff
@@ -83,22 +126,107 @@ Internally it uses a mask called a Camembert to generate a larger mask
 and limits it to the area affected by a line-darkening script.
 The main mask has no name and is simply dhhmask(mode=3).
 
-Example: Build a simple dehalo mask
+For more information about edgemasks, see [here](https://kageru.moe/blog/article/edgemasks).
+
+#### Example: Build a simple dehalo mask
+
+Suppose you want to remove these halos:
+
+![source](images/halos.png)
```

Comment:
Since the title will be used as the image caption, something more descriptive would be desireable. Same below.